share/templates.d/99-generic/live/config_files/aarch64/grub2-efi.cfg

```diff
@@ -26,16 +26,32 @@ set timeout=60
 search --no-floppy --set=root -l '@ISOLABEL@'
 
 ### BEGIN /etc/grub.d/10_linux ###
-menuentry 'Start @PRODUCT@ @VERSION@' --class red --class gnu-linux --class gnu --class os {
-	linux @KERNELPATH@ @ROOT@ @EXTRA@ rd.live.image quiet rhgb
+menuentry 'Start @PRODUCT@ @VERSION@' --class fedora --class gnu-linux --class gnu --class os {
```

Comment:
I think this should be changed to make it clear it will start with persistence.

Reply:
I'm not sure what to word here for what amounts to the default experience.

Reply:
How about `Start ... with automatic persistance`?